modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
 +----------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To restore a database to a write mode again after it has been lost due to a disaster.
+See xref:clustering/disaster-recovery.adoc[] for more information.
+ 
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Keep in mind the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on the store size and network speed.
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
+If only a subset of the previous cluster members is specified as seeders, there is a risk of data loss.
+To avoid this, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of the previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/[CDC documentation] for more details. - general link ?? Do we have any more specific guidelines on the topic?
```

Comment:
```suggestion
See link:https://neo4j.com/docs/cdc/current/[CDC documentation] for more details.
```

Reply:
This is a general link to the docs. Do we have any more specific guidelines on the topic?